targets/csharp/source/source/PlayFabHttp/PlayFabWinHttp.cs

```diff
@@ -10,6 +10,14 @@ namespace PlayFab.Internal
 {
     public class PlayFabWinHttp : IPlayFabHttp
     {
+		
+		private readonly HttpClient _client;
+
+        public PlayFabSysHttp()
+        {
+            _client = new HttpClient();
```

Comment:
Constructor not necessary. You can set this on line 14

Reply:
fixed